client/filter/tvs.geneVariant.js

```diff
@@ -166,55 +177,74 @@ async function fillMenu(self, _div, tvs) {
 		.style('margin', '10px')
 		.style('padding-left', '10px')
 		.style('text-align', 'left')
-		.each(function(grp) {
+		.style('opacity', 0.5)
+		.each(function (grp) {
 			const div = select(this)
-			div
-				.append('div')
-				.style('font-weight', 600)
-				.html(grp.name)
-			//.on('click', )
+			const dtTitleDiv = div.append('div').style('display', 'flex')
+			const grpEdited = exclude.some(e => grp.items.some(i => i.dt == e.dt && (i.origin ? i.origin == e.origin : true)))
+			const radioInput = dtTitleDiv
+				.append('input')
+				.attr('type', 'radio')
+				.attr('name', 'radioGroup')
+				.attr('value', grp.name)
+				.property('checked', grpEdited)
+				.on('click', () => {
+					dtDiv.style('opacity', 0.5)
+					div.style('opacity', 1)
+					dtDiv.selectAll('input[type="checkbox"]').property('disabled', true)
+					mClassDiv.selectAll('input[type="checkbox"]').property('disabled', false)
+				})
+			div.style('opacity', grpEdited ? 1 : 0.5)
 
-			div
+			const dtTitle = dtTitleDiv.append('div').style('font-weight', 600).html(grp.name)
```

Comment:
1. change it to LABEL element so can click for radio button change, rather than having to click on tiny button
2. need hover effect on LABEL 
3. by default, check a radio button with some non-WT, non-Blank data in it